src/timely-util/benches/reclock.rs

```diff
@@ -0,0 +1,129 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use std::sync::mpsc::Receiver;
+
+use criterion::{BenchmarkId, Criterion, criterion_group, criterion_main};
+use differential_dataflow::{
+    ExchangeData,
+    input::{Input, InputSession},
+};
+use mz_ore::Overflowing;
+use mz_timely_util::{capture::PusherCapture, order::Partitioned, reclock::reclock};
+use timely::{
+    communication::allocator::Thread,
+    dataflow::{
+        Scope,
+        operators::{
+            ActivateCapability, Capture, UnorderedInput, capture::Event,
+            unordered_input::UnorderedHandle,
+        },
+    },
+    progress::{Antichain, Timestamp, timestamp::Refines},
+    worker::Worker,
+};
```

Comment:
These imports use the default rust-analyzer style but we use individual `use` lines in the repo. I don't remember the details but apparently it's hard to convince rust-analyzer that this is the style we want for completely new files 😅